gui/src-tauri/src/main.rs

```diff
@@ -154,6 +214,58 @@ fn main() {
 			Ok(())
 		})
 		//
-		.run(tauri::generate_context!())
-		.expect("error while running tauri application");
+		.run(tauri::generate_context!());
+	match res {
```

Comment:
I added it for future convenience, I was going to add a check for a runtime error, but I would need ``tauri-runtime`` as a dependency

Reply:
No point preemptively doing something like that, just change it when you actually need it. For now since it makes it harder to read, let's revert that to how it was